lib/web_ui/lib/src/engine/canvaskit/canvaskit_api.dart

```diff
@@ -105,12 +106,9 @@ class CanvasKit {
   /// parameters specified in [SkImageInfo] passed [SkImage.readPixels] much
   /// match the arguments passed to this method.
   external SkImage MakeImage(
-    Uint8List imageData,
-    int width,
-    int height,
-    SkAlphaType alphaType,
-    SkColorType colorType,
-    ColorSpace colorSpace,
+    SkImageInfo info,
+    Uint8List pixels,
+    int bytesPerRow,
```

Comment:
The dartdoc above needs to be updated to reflect the new signature.

Reply:
Done.

Reply:
Done.